genomes/schemas/GenomeCatalogue.py

```diff
@@ -0,0 +1,29 @@
+from ninja import Schema
+from typing import Optional, Dict
+from datetime import datetime
+
+
+class GenomeCatalogueBase(Schema):
+    catalogue_id: str
+    version: str
+    name: str
+    description: Optional[str]
+    protein_catalogue_name: Optional[str]
+    protein_catalogue_description: Optional[str]
+    last_update: datetime
+    result_directory: Optional[str]
+    genome_count: Optional[int]
+    unclustered_genome_count: Optional[int]
+    ftp_url: str
```

Comment:
Maybe this is a dumb question... but can we use https://docs.pydantic.dev/2.1/api/networks/#pydantic.networks.AnyUrl